### DISCUSSION TITLE
low-level handling of HttpRequest in Resteasy Reactive

### DISCUSSION THREAD

**Discussion by kenyee** [quarkusio/quarkus#27951] [Q&A]
Is there any way to do low level handling of HttpRequest in a Resteasy Reactive endpoint?
Use case is needing to proxy the raw data from HttpRequest to another external API after adding some auth headers to it, then relaying the response from the API back to the HttpRequest (essentially a proxy).
Trying to understand how to handle this as a Uni<> response w/o needing to know the types of data being sent back and forth.

[a comment by an automated account is not shown]

**Comment by geoand** [comment]
Hi,

For the use case you mention, I would stronly suggest looking into [Vert.x HTTP Proxy](https://vertx.io/docs/vertx-http-proxy/java/)
